package.json

```diff
@@ -2,7 +2,8 @@
   "name": "britecharts-react",
   "version": "1.0.0",
   "description": "Britecharts-react is a React wrapper for the Britecharts charting library",
-  "main": "index.js",
+  "main": "dist/britecharts-react.min.js",
```

Comment:
`main` is used by Node so I don't think you'd want to point it to the `dist` minified bundle. You'd want it to pointing to unminified CommonJS (`lib/umd`?)

Reply:
Thanks! I will correct it now!